scripts/ci-entrypoint.sh

```diff
@@ -22,11 +22,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Install kubectl and helm
+# Install kubectl, helm and kustomize
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 KUBECTL="${REPO_ROOT}/hack/tools/bin/kubectl"
 HELM="${REPO_ROOT}/hack/tools/bin/helm"
-cd "${REPO_ROOT}" && make "${KUBECTL##*/}"; make "${HELM##*/}"
+KUSTOMIZE="${REPO_ROOT}/hack/tools/bin/kustomize"
+
+make --directory="${REPO_ROOT}" "${KUBECTL##*/}" "${HELM##*/}" "${KUSTOMIZE##*/}"
+
 # export the variables so they are available in bash -c wait_for_nodes below
 export KUBECTL
```

Comment:
ci-entry does `bash -c` below in the function wait for nodes which is why we export it here.   It isn't used in the function below so I don't think it is needed here but fyi

Reply:
Thank you for the update!